src/wled2.ts

```diff
@@ -48,13 +47,102 @@ export = (RED: Red): void => {
     });
   });
 
+  // This function will 
+  function which(
+    incomingPayloadValue: string | number | boolean | IWledSegment[],
+    nodePayloadValue: string | number | boolean | IWledSegment[],
+    wrapper?: NumberConstructor // Add more types if used for others later.
+  ): string | number | boolean | IWledSegment[] {
+    const val = incomingPayloadValue ?? nodePayloadValue ?? undefined;
+    return wrapper && val !== undefined ? wrapper(val) : val
+  }
+

```

Comment:
This just doesn't seem like it adds any code clarity to the `getMergedPayloadAndConfig()` function and makes things messier with a bunch of type casting and variable type parameters.

Given the elegance of the `??` operator this method can just go away and the getMergedPayloadAndConfig method can simply directly do the single line that's the meat of this method (see below). It's also weird to have it handle the casting to Number since that's the only type that has to get cast, and only happens in a few instances generally,

Reply:
I was originally using it with the color helpers as well, but moved away from that. This is the only way to get around the lint rule without disabling it that I could personally get to work.